src/main/views/manage-user.njk

```diff
@@ -22,6 +22,17 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-three-quarters">
       {{ errorSummary(error) }}
+      {% if content.result %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: [
+            {
+              text: "content.result",
+              href: "#search"
+            }
+          ]
+        }) }}
+      {% endif %}
```

Comment:
At the end of this file, the following lines can probably be removed now you don't set the warning text anymore:

{% if content.result %}
        {{ govukWarningText({
          text: content.result,
          iconFallbackText: "Warning"
        }) }}
      {% endif %}

Reply:
Isn't this a duplicate of the errorSummary macro just above? That adds an error summary to the page if there's an errors passed to the view

Reply:
I think the error is set in 2 places now. The error summary at the top, and the error message on the input text box (like the other errors). The warning text is not required anymore because "content.result" is not passed in and will never be set. Imran has removed that in the latest commit. @Thomas-Geraghty I am not sure what u mean by 'duplicate of errorSummary'?

Reply:
oic. I see what u mean by duplication of error summary :)

Reply:
i believe you are right it seems to still work if i take my bit out